examples/official-site/sqlpage/migrations/01_documentation.sql

```diff
@@ -739,6 +739,10 @@ and `shell.json` would be placed at the website''s root and contain the followin
 INSERT INTO component(name, icon, description) VALUES
     ('shell', 'layout-navbar', 'Personalize the "shell" surrounding your page contents. Used to set properties for the entire page.');
 
+INSERT INTO parameter(component, name, description_md, type, top_level, optional) SELECT 'shell', * FROM (VALUES
+    ('favicon', 'The URL of the icon the web browser should display in bookmarks and tabs. This property is particularly useful if multiple sites are hosted on the same domain with different [``site_prefix``](https://github.com/lovasoa/SQLpage/blob/main/configuration.md#configuring-sqlpage).', 'URL', TRUE, TRUE),
```

Comment:
I think we render the descriptions here as text, not markdown. But we have markdown and syntax highlighting in the examples below, where we can be a little bit more verbose